paasta_tools/kubernetes_tools.py

```diff
@@ -2398,7 +2387,7 @@ async def get_kubernetes_app_deploy_status(
         deploy_status = KubernetesDeployStatus.Stopped
     else:
         deploy_status = KubernetesDeployStatus.Running
-    return deploy_status, deploy_message
```

Comment:
can we not just make it return one thing?  or is that too hard because the cli is expecting something...?

At the very least can we leave a TODO with an explanation?

Reply:
Yeah, there are a few function calls in between - I just didn't want to edit them all if we are going to bring the deploy message back. Added a comment.